blocks/leaphy_common.js

```diff
@@ -431,6 +431,38 @@ function getBlocks(board) {
       tooltip: "",
       helpUrl: "",
     },
+    {
+      type: "leaphy_gas_sensor",
+      message0: "Choose gas: %1",
```

Comment:
Kun je voor "Choose gas" nog een vertaling toevoegen (en er een translation tag van maken)?

Reply:
Ik heb de vertaling toegevoegd. Wist alleen niet zo goed meer hoe je de translations kan updaten?

Reply:
In principe is toevoegen aan de .js genoeg, maar wel netjes om ook de .json te updaten. En kun je nog de nl vertaling toevoegen?